nservicebus/upgrades/support-policy.md

```diff
@@ -9,7 +9,7 @@ related:
 
 ## Supported Versions
 
-Supported versions include any major version released within the last 3 years. Within a supported major version, the latest minor version is supported, or any minor version released within the last year.
+Supported versions include any major version released within the last 3 years. Within a supported major version, the latest minor version is supported. For the NServiceBus package only (i.e. excluding downstream packages such as [transports](/nservicebus/transports/) and [persistence](/nservicebus/persistence/)), any minor version released within the last year is supported.
```

Comment:
@indualagarsamy see https://github.com/Particular/Lisbon2017.Unconference/issues/16#issuecomment-275626656

I guess this does need some work to make it clearer. Perhaps a table would be better?